karbon/v1alpha1/k8s_registration_structs.go

```diff
@@ -0,0 +1,102 @@
+package v1alpha1
+
+type K8sCreateClusterRegistrationResponse struct {
+
+	// cluster name
+	// Required: true
+	ClusterName *string `json:"cluster_name"`
+
+	// cluster uuid
+	// Required: true
+	ClusterUUID *string `json:"cluster_uuid"`
+
+	// task uuid
+	// Required: true
+	TaskUUID *string `json:"task_uuid"`
```

Comment:
we need sample for how to wait on the taskUUID passed here as return value to check the status of registration

Reply:
@deepakm-ntnx so you mean that we need to add the wait the task in the CreateK8sRegistration api under prism-go-client? I think we could add in the prism-go-client instead of in the CLI for the check. The task UUID could be used to check the status with timeout to see if the creation succeed. But one thing that I am thinking is that since the K8sCreateClusterRegistrationResponse has been successfully received with the taskUUID the creation task should be succeed so this check may not be a must needed at this time.